src/host/ut_host/TextBufferTests.cpp

```diff
@@ -402,29 +409,26 @@ void TextBufferTests::TestInsertCharacter()
     // create some sample test data
     const auto wch = L'Z';
     const std::wstring_view wchTest(&wch, 1);
-    DbcsAttribute dbcsAttribute;
-    dbcsAttribute.SetTrailing();
+    const auto dbcsAttribute = DbcsAttribute::Leading;
     const auto wAttrTest = BACKGROUND_INTENSITY | FOREGROUND_INTENSITY | FOREGROUND_RED | FOREGROUND_BLUE;
     auto TestAttributes = TextAttribute(wAttrTest);
 
-    auto& charRow = Row.GetCharRow();
-    charRow.DbcsAttrAt(coordCursorBefore.X).SetLeading();

```

Comment:
Whether something is leading or trailing is now entirely inferred by whether something is actually occupying 2 cells or not and so you can't `SetLeading()` anymore either because `DbcsAttrAt()` now returns an immutable view on what the text buffer contains at that position.